src/platform/src/Bridge/Gemini/Gemini/ResultConverter.php

```diff
@@ -189,6 +189,6 @@ private function isSuccessfulCodeExecution(array $contentPart): bool
 
         $result = $contentPart['codeExecutionResult'];
 
-        return 'OUTCOME_OK' === $result['outcome'];
+        return CodeExecutionResultOutcome::OK->value === $result['outcome'];
```

Comment:
```suggestion
        return CodeExecutionResultOutcome::from($result['outcome'])->equals(CodeExecutionResultOutcome::OK);
```
after using the Comparable trait